cypress/BUILD.cypress

```diff
@@ -16,11 +18,17 @@ copy_directory(
     }),
 )
 
+copy_file(
+    name = "binary_state_file",
+    src = "binary_state_temp.json",
```

Comment:
You can use the `copy_to_bin` rules from `aspect_bazel_lib` to get around the src/out issue.

```
copy_to_bin(
   name = "binary_state_file",
   srcs = ["binary_state.json"],
)
```

Reply:
That rule copies a source file to the output tree without changing its name

Reply:
changed!